ui/src/authors/containers/DetailPageContainer/DetailPageContainer.jsx

```diff
@@ -145,23 +152,51 @@ function DetailPage({
                 </Row>
               </ContentBox>
             </Col>
-            <Col xs={24} md={12} lg={8}>
-              <ContentBox subTitle="Citations per year">
-                <EmptyOrChildren data={publications} title="0 Research works">
-                  <CitationsByYearGraphContainer />
-                </EmptyOrChildren>
-              </ContentBox>
+          </Row>
+          <Row>
+            <Col span={24}>
+              <Tabs type="card" tabBarStyle={{ marginBottom: 0 }}>
```

Comment:
probably this `Tabs` pattern can be a component like `ContentBoxTabs`, same thing exist in literature detail page too.

Reply:
But that wouldn't simplify it that much, right? We'd need a `ContentBoxTabs` which should then have as children a `ContentTab` or something like that which gets as props `tooltip`, `name`, `count`, `key`, `forceRender`, `children`...

Reply:
only for `Tabs` that will have to style which you copy pasted from literature detail page to here too, and these props. it won't do anything else, basically ` <Tabs ...props and class name...>{children}</Tabs>`

Reply:
It would only extract the `type="card"` and `tabBarStyle`. `className` is set to `Tabs` only in literature. In authors I added it individually to the `ContentBox` in each tab since otherwise it'd also remove the top border of the cards inside the tab (which we don't have in literature).

Reply:
I guess it's not difficult make a style that works both though.. but on the other hand why do we need ContentBox for each tab? we can just wrap whole `Tabs` maybe?

Reply:
Yeah we can with an uglier selector like `.remove-top-border-of-card > .ant-tabs-content > .ant-tabs-tabpane > .__ContentBox__ > .ant-card-bordered`. We could use a `Card` instead of a `ContentBox` but then we'd need to tweak the margins and so. How do you want to wrap it anyway?

Reply:
wrapping that I meant.
```
<ContentBox>
  <Tabs..
</ContentBox>
```

from there, you probably need less css to make it look like we want. I can't see exactly without playing with it locally but give it a try, if not that's OK too I guess.

Reply:
If I do this then the tabs are inside the white `ContentBox` and they look super weird :(